extern/bullet2/src/BulletCollision/CollisionShapes/btConeShape.h

```diff
@@ -43,6 +43,15 @@ ATTRIBUTE_ALIGNED16(class) btConeShape : public btConvexInternalShape
 	btScalar getRadius() const { return m_radius;}
 	btScalar getHeight() const { return m_height;}
 
+	virtual void setRadius(btScalar radius)
```

Comment:
Hi! I made the same comment for TwisterGE's patch about vehicle speed: I don't know if this is a good idea to change bullet sources as it's an external lib and that this will create conflict when we'll switch to another bullet version.
Maybe there is another way to design this patch without modifying bullet sources? @panzergame: What do you think?